src/rpc/index.js

```diff
@@ -23,7 +22,7 @@ const { AES, RSA, SHA1, SHA256 } = require('../utils/crypto');
 const { getRsaKeyByFingerprints } = require('../utils/rsa');
 
 class RPC {
-  constructor({ api_id, api_hash, dc, updates }) {
+  constructor({ api_id, api_hash, dc, updates, storage }) {
```

Comment:
cant you just give the two things this thing needs? 
```suggestion
  constructor({ api_id, api_hash, dc, updates, authKey, serverSalt }) {
```